stream-product-catalog/product-catalog-core/src/main/java/com/backbase/stream/productcatalog/ReactiveProductCatalogService.java

```diff
@@ -170,7 +159,7 @@ public Mono<ProductType> createProductType(ProductType productType, List<Product
 
                 arrangementProductItemBase.setExternalProductKindId(productKind.getExternalKindId());
                 arrangementProductItemBase.setProductKindName(productKind.getKindName());
-                arrangementProductItemBase.setTypeName(productType.getTypeName());
+                arrangementProductItemBase.setProductTypeName(productType.getTypeName());
```

Comment:
We already have both `TypeName` and `ProductTypeName` in stream's [OpenAPI contract](https://engineering.backbase.com/stream-services/api/stream-product-catalog/index.html#tag/product-catalog/operation/setupProductCatalog), I am not sure why you want to map `TypeName` to `ProductTypeName`?

Reply:
The field `productTypeName` is used in `arrangement-manager` to populate the `type_name` column. While I agree that having both `typeName` and `productTypeName` creates some redundancy, changing it to the current setup (`typeName`, introduced during the `service-api` `v3` update) would require altering how `arrangement-manager` handles the API. This adjustment would result in a breaking change, as it previously relied on `productTypeName`.